benchs/tpch/run.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -eu
+set -o pipefail
+
+TAR_VER=$(tarantool -v | grep -e "Tarantool" |  grep -oP '\s\K\S*')
+numaopts="numactl --membind=1 --cpunodebind=1 --physcpubind=6,7,8,9,10,11"
+base_dir=$(pwd)
+
+cd /opt/tpch
+make TPC-H.db
+
+killall tarantool tpcc_load 2>/dev/null || true
+sync && echo "sync passed" || echo "sync failed with error" $?
+echo 3 > /proc/sys/vm/drop_caches
+
+$numaopts /opt/tpch/bench_queries.sh 2>&1 | tee bench-sqlite.log
```

Comment:
This is doing almost exactly the same as if you call `make bench-sqlite` but with numa options overriden. I'm ok to start with this current approach in this patch, but we should rather consider a way to fine-tune `Makefile` - to keep all internals there.
(One fine day we may change script name or add options or whatever, and this direct all would be broken since then. But no iff you would call make target)

Reply:
I have done changes into tpch Makefile to keep all internals there